src/net_processing.cpp

```diff
@@ -3862,6 +3965,17 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         return;
     }
 
+    if (pfrom.IsPrivateBroadcastConn()) {
+        if (msg_type != NetMsgType::PONG && msg_type != NetMsgType::GETDATA) {
+            LogDebug(BCLog::PRIVBROADCAST,
+                     "Ignoring incoming message '%s', peer=%d%s",
+                     msg_type,
+                     pfrom.GetId(),
+                     pfrom.LogIP(fLogIPs));
+            return;
+        }
+    }
```

Comment:
If you think the other type helper was useful, maybe we could reduce the indendation here, too:
```C++
    if (pfrom.IsPrivateBroadcastConn() && !IsPrivateBroadcastInboundAllowed(msg_type)) {
        LogDebug(BCLog::PRIVBROADCAST,
                 "Ignoring incoming message '%s', peer=%d%s",
                 msg_type,
                 pfrom.GetId(),
                 pfrom.LogIP(fLogIPs));
        return;
    }
```
via
```C++
bool IsPrivateBroadcastInboundAllowed(std::string_view msg_type) noexcept
{
    return msg_type == NetMsgType::PONG
        || msg_type == NetMsgType::GETDATA;
}
```

These helpers could of course be put to a place that's easy to review - otherwise the logic is spread around.